wlct/tournaments.py

```diff
@@ -2422,12 +2422,19 @@ def finish_game(self):
         self.is_finished = True
 
         self.current_state = "Finished"
+        self.tournament = find_tournament_by_id(self.tournament.id)
```

Comment:
please create a swiss or seeded tournament and hit start, make sure it looks good at the end with no crashes.

Reply:
No crashes, but some columns seem to be unimplemented:

- Swiss tournament
  - Tournament game log
- Seeded tournament
  - Tournament game log
  - Bracket/Games

Reply:
Columns in the database? Or on the actual page?

Reply:
On the actual page, the tabs are blank

Reply:
For swiss I think that's OK, for seeded the bracket should be present. That is probably another bug...ugh...ok I will take a look at this tomorrow then but it sounds like there are some issues generating this data then.